src/components/common/FetchHandler.js

```diff
@@ -23,4 +23,4 @@ class FetchHandler {
     });
 }
 
-export default FetchHandler;
+export default FetchHandler;
```

Comment:
```suggestion

export default FetchHandler;
```
Keep trailing newline